### PR TITLE
[update][m] Adding necessary changes to files

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made
- Added `.gitignore` file
- Polished the `README.md` file
- Updated the `datapackage.json` file
- Updated the `data/` folder

cc @anuveyatsu

### CODE REVIEW

File: datapackage.json [hunk 1]
```diff
@@ -1,116 +1,80 @@
...
         "fields": [
           {
             "name": "Source",
             "type": "string"
           },
           {
-            "description": "YYYY",
             "name": "Year",
-            "type": "year"
+            "type": "date",
```
Comment (anuveyatsu) [inline datapackage.json:30]: @gradedSystem I don't believe it is actually "date" type as default format would be `YYYY-MM-DD`. You need to either set type as `year` or specify the format.

File: datapackage.json [hunk 2]
```diff
@@ -1,116 +1,80 @@
...
             "name": "Source",
             "type": "string"
           },
           {
-            "description": "YYYY-MM",
-            "format": "any",
             "name": "Date",
-            "type": "date"
+            "type": "date",
+            "description": "YYYY-MM"
```
Comment (anuveyatsu) [inline datapackage.json:55]: @gradedSystem please add `"format": "any"` as it is needed when performing data validation using this table schema.

File: datapackage.json [hunk 3]
```diff
@@ -1,116 +1,80 @@
...
+      "id": "graph",
+      "label": "Graph",
+      "type": "Graph",
+      "state": {
         "group": "Year",
         "series": [
           "Mean"
-        ]
+        ],
+        "graphType": "columns"
```
Comment (anuveyatsu) [inline datapackage.json:76]: @gradedSystem I'm not sure if we have this type and generally if this is correct spec. I would rather copy the old view spec.

File: data/annual.csv [hunk 4]
```diff
@@ -1,275 +1,320 @@
...
-GCAG,1883,-0.1424
-GISTEMP,1883,-0.21
-GCAG,1882,-0.0648
-GISTEMP,1882,-0.1
-GCAG,1881,-0.0628
-GISTEMP,1881,-0.12
-GCAG,1880,-0.1148
-GISTEMP,1880,-0.2
+Source,Year,Mean
+gcag,1850,-0.4177
```
Comment (anuveyatsu) [inline data/annual.csv:2]: @gradedSystem can we keep source name in upper case as in the old data file - `GCAG`? Same applied to the monthly data file.

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
@gradedSystem I left some comment, pls, take a look 🙏🏼

File: data/annual.csv (hunk 4 above)
Comment (gradedSystem) [reply data/annual.csv:2]: Yes I forgot to write it in UPPERCASE

**Comment by gradedSystem** [comment]
I am going to merge this PR for now and update the script with the UPPERCASE Data column

**Pull request merged**